test_vm/vendor/cookbooks/dbfit_test/recipes/diskspace.rb

```diff
@@ -0,0 +1,19 @@
+chef_gem 'di-ruby-lvm'
+
+lvm_physical_volume '/dev/sdb' do
+  action :create
+end
+
+lvm_volume_group 'centos' do
+  action :extend
+  physical_volumes ['/dev/sdb']
+end
+
+lvm_logical_volume 'root' do
+  group 'centos'
+  action :resize
+  size '9G'
+  filesystem 'ext2'
```

Comment:
Is it really ext2? I would rather expect ext3 or ext4. Is that parameter needed for resize?